src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -118,6 +118,14 @@ services:
         arguments: [!iterator ['@foo.baz', '@?invalid'], !iterator []]
     alias_for_foo: '@foo'
     alias_for_alias: '@foo'
+    tagged_iterator_foo:
+        class: Bar
+        tags:
+            - { name: foo }
+    tagged_iterator:
+        class: Bar
+        arguments:
+            - !tagged_iterator foo
```

Comment:
Once again, you have to do https://github.com/symfony/symfony/pull/22200#issuecomment-290128785 ;)